_pages/activities/gsoc.md

```diff
@@ -8,13 +8,17 @@ sidebar:
 classes: wide
 
 
-gsoc2024:
+gsoc2025:
+  - url: /activities/gsoc/2025
+    image_path: /assets/images/activities/gsoc/gsoc-2024.jpg
```

Comment:
We could add the image from https://summerofcode.withgoogle.com/programs/2025-ao

![GSoC image](https://summerofcode.withgoogle.com/assets/media/gsoc-generic-badge.svg)